server/netty-server/loom/src/test/scala/sttp/tapir/server/netty/loom/SleepDemo.scala

```diff
@@ -0,0 +1,11 @@
+package sttp.tapir.server.netty.loom
+
+import sttp.tapir._
+
+object SleepDemo extends App {
```

Comment:
this can be removed now?

Reply:
It doesn't seem to add anything more than it's in the docs, so I'll just remove it.